### PR TITLE
[HDF5] Adding examples to ReadMe file

### PR DESCRIPTION
**Pull request by sunethwarna**
**Description**
Adds process example json settings.

**Changelog**
- Added example process json settings

### CODE REVIEW

File: applications/HDF5Application/README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+  - [Note](#note)
+  - [Kratos processes](#kratos-processes)
+    - [Initialization from hdf5 process](#initialization-from-hdf5-process)
+    - [Single mesh temporal input process](#single-mesh-temporal-input-process)
+    - [Single mesh temporal output process](#single-mesh-temporal-output-process)
+    - [Multiple mesh temporal output process](#multiple-mesh-temporal-output-process)
+    - [Single mesh xdmf output process for Paraview](#single-mesh-xdmf-output-process-for-paraview)
+    - [User defined I/O process](#user-defined-io-process)
+
 # HDF5Application
```
Comment (philbucher) [inline applications/HDF5Application/README.md]: I would move the overview after the introduction section, is nicer to read IMO

**Review by philbucher — COMMENTED** [review]
minor otherwise nice

File: applications/HDF5Application/README.md (hunk 1 above)
Comment (sunethwarna) [reply applications/HDF5Application/README.md]: done :)

**Review by philbucher — APPROVED** [review]
cool!

**Pull request merged**